src/System.Net.Security/src/System/Net/NTAuthentication.cs

```diff
@@ -429,13 +341,11 @@ internal byte[] GetOutgoingBlob(byte[] incomingBlob, bool throwOnError, out Inte
                 if (!_isServer)
                 {
                     // client session
-                    statusCode = (Interop.SecurityStatus)SSPIWrapper.InitializeSecurityContext(
-                        GlobalSSPI.SSPIAuth,
+                    statusCode = NegoState.InitializeSecurityContext(
```

Comment:
Please refer to the latest commits in #5772 and see if that answers your question. As to why, NegoState uses NTAuthentication, that is the original design that has been just ported so we won't have insight into it.